src/webgpu/api/validation/resource_usages/textureUsageInPassEncoder.spec.ts

```diff
@@ -797,17 +893,40 @@ g.test('bindings_in_bundle')
 g.test('unused_bindings_in_pipeline')
   .params(
     params()
+      // Test compute pass if 'compute' is true. Test render pass otherwise.
```

Comment:
I'm not sure that it helps to have that comment copy-pasted everywhere. Maybe an explanation at the top of the file would be more helpful.

Reply:
Done.